os-config_arm.yml

```diff
@@ -0,0 +1,328 @@
+        io.rancher.os.detach: "false"
+        io.rancher.os.scope: system
+        io.rancher.os.after: console
+      privileged: true
+      volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+      - /var/lib/docker/preload:/mnt/preload
+      volumes_from:
+      - command-volumes
+      - system-volumes
+    syslog:
+      image: rancher/os-syslog:v0.4.3-dev_arm
+      labels:
+        io.rancher.os.scope: system
+      log_driver: json-file
+      net: host
+      uts: host
+      privileged: true
+      restart: always
+      volumes_from:
+      - system-volumes
+    system-volumes:
+      image: rancher/os-state:v0.4.3-dev_arm
+      labels:
+        io.rancher.os.createonly: "true"
+        io.rancher.os.scope: system
+      log_driver: json-file
+      net: none
+      privileged: true
+      read_only: true
+      volumes:
+      - /dev:/host/dev
+      - /etc/docker:/etc/docker
+      - /etc/hosts:/etc/hosts
+      - /etc/resolv.conf:/etc/resolv.conf
+      - /etc/rkt:/etc/rkt
+      - /etc/ssl/certs/ca-certificates.crt:/etc/ssl/certs/ca-certificates.crt.rancher
+      - /lib/firmware:/lib/firmware
+      - /lib/modules:/lib/modules
+      - /run:/run
+      - /usr/share/ros:/usr/share/ros
+      - /var/lib/rancher/conf:/var/lib/rancher/conf
+      - /var/lib/rancher:/var/lib/rancher
+      - /var/log:/var/log
+      - /var/run:/var/run
+    udev-cold:
+      image: rancher/os-udev:v0.4.3-dev_arm
+      labels:
+        io.rancher.os.scope: system
+        io.rancher.os.before: udev
+      net: host
+      uts: host
+      privileged: true
+      volumes_from:
+      - system-volumes
+    udev:
+      image: rancher/os-udev:v0.4.3-dev_arm
+      environment:
+      - DAEMON=true
+      labels:
+        io.rancher.os.detach: "true"
+        io.rancher.os.scope: system
+      net: host
+      uts: host
+      privileged: true
+      restart: always
+      volumes_from:
+      - system-volumes
+    user-volumes:
+      image: rancher/os-state:v0.4.3-dev_arm
+      labels:
+        io.rancher.os.createonly: "true"
+        io.rancher.os.scope: system
+      log_driver: json-file
+      net: none
+      privileged: true
+      read_only: true
+      volumes:
+      - /home:/home
+      - /opt:/opt
+    docker:
+      image: rancher/os-docker:v0.4.3-dev_arm
+      labels:
+        io.rancher.os.scope: system
+        io.rancher.os.after: console
+      net: host
+      pid: host
+      ipc: host
+      uts: host
+      privileged: true
+      restart: always
+      volumes_from:
+      - all-volumes
+      volumes:
+      - /sys/fs/cgroup:/host/sys/fs/cgroup
+  system_docker:
+    args: [daemon, --log-opt, max-size=25m, --log-opt, max-file=2, -s, overlay, -b, docker-sys,
+      --fixed-cidr, 172.18.42.1/16, --restart=false, -g, /var/lib/system-docker, -G, root,
+      -H, 'unix:///var/run/system-docker.sock', --userland-proxy=false]
+  upgrade:
+    url: https://releases.rancher.com/os/releases_arm.yml
+    image: rancher/os
```

Comment:
I also meant that the name of the image might change to `rancher/os-arm` or something. Not so important right now but anyway